cli/src/commands/file/annotate.rs

```diff
@@ -51,6 +52,22 @@ pub(crate) struct FileAnnotateArgs {
         add = ArgValueCandidates::new(complete::all_revisions)
     )]
     revision: Option<RevisionArg>,
+    /// Render each line using the given template
+    ///
+    /// Run `jj log -T` to list the built-in templates.
+    ///
+    /// You can also specify arbitrary [template expressions] using the
+    /// [built-in keywords].
+    ///
+    /// If not specified, this defaults to the
+    /// `templates.annotate_commit_summary` setting.
+    ///
+    /// [template expression]:
+    ///     https://jj-vcs.github.io/jj/latest/templates/
+    /// [built-in keywords]:
+    ///     https://jj-vcs.github.io/jj/latest/templates/#commit-keywords
```

Comment:
nit: insert blank line so these link anchors won't be reformatted into a single line